src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -195,12 +198,12 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
   }
 
   /**
-   * Attempt to start all runnable calls and return the next FSM state.  If successful this will be
-   * `WorkflowRunning`, otherwise `WorkflowFailed`.
+   * Attempt to start all runnable calls and return the next FSM state.  If successful this will
+   * `stay()` in `WorkflowRunning`, otherwise `WorkflowFailed`.
    */
   private def startRunnableCalls(): State = {
     tryStartingRunnableCalls() match {
-      case Success(entries) => if (entries.nonEmpty) startRunnableCalls() else transition(WorkflowRunning)
+      case Success(entries) => if (entries.nonEmpty) startRunnableCalls() else stay()
```

Comment:
So this happens because we now assume tryStartRunnableCalls only gets called from the WorkflowRunning state? Can that be made explicit in the comment (or else checked for) in case future-someone tries to call this from elsewhere?